### DISCUSSION TITLE
Name for the Application

### DISCUSSION THREAD

**Discussion by PabloGOP** [Arquisoft/wiq_es04a#19] [Ideas]
In our pursuit of creating a globally recognized web app inspired by the renowned Spanish game show "Saber y Ganar," it's imperative that we select a name that resonates powerfully on an international scale. As such, I propose **"Wikidata Inquisition Quest"** as our app's name. This name not only highlights our reliance on Wikidata for trivia questions but also evokes a sense of adventure and intrigue, making it memorable and appealing to a wide audience. Let's discuss further to ensure our app stands out on the global stage and that we all are aligned in this issue.

**Comment by alagoconde** [comment]
This idea makes sense to me so I obviously support it, but I think your proposed name could be better, or at least more serious, maybe something like Wikidata Intelligence Quiz?

**Comment by and1na** [comment]
What a great idea! I think it's fantastic to add a touch of personality to the application. Since our main strength lies in generating a large number of questions with updated answers, my proposed names are **'Wiki Infinite Quest' or 'Wiki Infinite Questions.'** We should conduct a vote in the upcoming meeting.

**Reply by alagoconde** [reply]
As discussed in the Meeting Minute [MM 04 - Remote 02](https://github.com/Arquisoft/wiq_es04a/wiki/MM-04-%E2%80%90-Remote-02), the name proposed in this comment has been adapted to "Wikidata Infinite Quest" and chosen as the provisional application' name.
